assets/compat.ini

```diff
@@ -802,10 +802,10 @@ ULJS00119 = true
 ULKS46167 = true
 NPJH50017 = true
 
-[ForceMax60FPS]
-# The GOW games are very heavy and render as fast as they can. They benefit greatly from
-# capping the framerate at 60fps.
-
+[Fixed60FPShack]

```

Comment:
Let's rename this GoWFramerateHack60 or something like that, similarly for Fixed30FPShack.